phabricatoremails/render/template.py

```diff
@@ -0,0 +1,268 @@
+import textwrap
+from datetime import datetime, timezone
+from typing import Dict
+
+import jinja2
+from phabricatoremails.render.events.common import Reviewer, ReviewerStatus
+from phabricatoremails.render.events.phabricator import (
+    ReplyContext,
+    CodeContext,
+    DiffLineType,
+    AffectedFileChange,
+    ExistenceChange,
+)
+from premailer import Premailer
+
+
+EMOJI = {
+    "airplane": "2708",
+    "airplane_arrival": "1f6ec",
+    "bell": "1f514",
+    "check_mark": "2714",
+    "circle_arrows": "1f504",
+    "deny_x": "2716",
+    "gear": "2699",
+    "keyboard": "2328",
+    "loudspeaker": "1f4e2",
+    "memo": "1f4dd",
+    "microscope": "1f52c",
+    "reply": "21a9",
+    "speech_balloon": "1f4ac",
+    "wrench": "1f527",
+}
+
+
+def emoji_html(key: str):
```

Comment:
Since this function is only used within the template, I'm optimizing for ergonomics there.
I don't want templates to have to:
```
{{ EMOJI["airplane"] | emoji | safe }}
```
Since that's a little tedious. 

> You can also add functionality to parse this to a hex value as well, if needed.

Isn't the result of `EMOJI[name]` a hex value already? :thinking: I'm not quite sure what you mean